_content/tour/por/goroutines.article

```diff
@@ -368,24 +273,24 @@ What happens if you go back to the original program but change GOMAXPROCS so the
 What you see is that the concurrency of the program is now more fine grained. The output
 to the letter is undefined and out of order.
 
-** Notes
+** Notas
 
-- Goroutines are functions that are scheduled to run independently.
-- We must always maintain an account of running goroutines and shutdown cleanly.
-- Concurrency is not parallelism.
+- Goroutines são funções agendadas para serem executadas independentemente.
+- Devemos sempre manter um registro das goroutines em execução e encerrar de forma limpa.
+- Concorrência não é paralelismo.
 
-- Concurrency is about dealing with lots of things at once.
-- Parallelism is about doing lots of things at once.
+- Concorrência trata de lidar com muitas coisas ao mesmo tempo.
+- Paralelismo trata de fazer muitas coisas ao mesmo tempo.
 
-"Parallelism is about physically doing two or more things at the same time. Concurrency is about undefined, out of order, execution." - William Kennedy
+"Paralelismo trata de fazer fisicamente duas ou mais coisas ao mesmo tempo. Concorrência trata de uma execução indefinida e fora de ordem." - William Kennedy
```

Comment:
If you chage up there, change here too!

Reply:
I didn't understand this comment.

Reply:
This comment is abot changing the "execução indefinida e fora de ordem" if you change in the other comment, but I think it's fine how you translated.